CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## 1.1.1 - [#5](https://github.com/openfisca/extension-template/pull/5)
+
+* Technical improvement: adapt to version `17.0.0` of Openfisca-Core and version `1.2.4` of Country-Template
```

Comment:
In my opinion, this PR is a fix (same features) consisting in adapting to the new version of openfisca-core. This implies the transformation from XML to YAML, but this is not the main concern of the user of the package.
If somebody else shares your opinion I will change the title.